src/loader.cpp

```diff
@@ -59,14 +61,31 @@ loaded_morphology load_asc(const std::filesystem::path &fn) {
     return result;
 }
 
+loaded_morphology load_acc(const std::filesystem::path &fn) {
+    if (auto m = arborio::parse_component(slurp(fn))) {
+       try {
+           return loaded_morphology{.morph=std::get<arb::morphology>(m.value().component)};
+       }
+       catch (std::bad_variant_access const& ex) {
```

Comment:
Here, we probably should bubble up to the user and tell them that we couldn't load.